test_vae_training/1_train_vae.py

```diff
@@ -0,0 +1,86 @@
+# ---
+# jupyter:
+#   jupytext:
+#     formats: ipynb,py
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.9.1+dev
+#   kernelspec:
+#     display_name: Python [conda env:microbe] *
+#     language: python
+#     name: conda-env-microbe-py
+# ---
+
+# # Train VAE model
+#
+# This notebook will first try to train the current VAE model before modifying the loss function to work with count data
+
+# %load_ext autoreload
+# %autoreload 2
+# %matplotlib inline
+import os
+import matplotlib.pyplot as plt
+import pandas as pd
+from ponyo import utils, train_vae_modules
+
+# Set seeds to get reproducible VAE trained models
+train_vae_modules.set_all_seeds()
```

Comment:
How much of a pain is it to switch to the new function calls being shown in the ipynb warnings? Could make the document a little easier to read.

Reply:
So unfortunately pretty painful:) So `tf.compat.v1.ConfigProto` was introduced in TF v2.0 and currently ponyo is running on ponyo v <2.0 https://stackoverflow.com/questions/56127592/attributeerror-module-tensorflow-has-no-attribute-configproto

I have tried playing around trying to update TF for security reasons but I wasn't able to update it to v2.0 as other dependencies broke. 

I plan to update ponyo to use pytorch later so hopefully this will not be a headache for anyone trying to use ponyo and modify it later.